### ISSUE TITLE
Notify is available in 0.6.0 ? 

### ISSUE THREAD

**Issue by Nayir** [vuikit/vuikit#40]
Hi,

I was wondering if the new feature Notify is available. I don't know how to check manually. Maybe you can teach me how for the next feature :)

**Comment by vuchl** [comment]
@miljan-aleksic [pushed the notify component later](https://github.com/vuikit/vuikit/commit/dad08545836f51292326babb03ac9cadf1a233f9) to the develop branch then the 0.6.0 commit. So no.

**Comment by miljan-aleksic** [comment]
@Nayir, all new features in dev branch will be available as 0.7.0 once considered stable enough. Meantime you can build manually and install Vuikit from your local repo. Something like this:

```
// in your vuikit local repo folder
npm install
npm run build

// in your vue project folder
npm install /local/path/to/vuikit/repo
```

**Comment by Nayir** [comment]
Great. Thanks
